templates/boards/topic_detail.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html' %}
```

Comment:
Format all your HTML files. Indentation should be 2 spaces. And remove unnecessary line breaks. This you can commit to master directly for all the template files. Ignore in this PR.